### PR TITLE
apache-spark: fix a test for the dash

### PR DESCRIPTION
**Pull request by tmozgach**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/apache-spark.rb [hunk 1]
```diff
@@ -18,6 +18,6 @@ def install
   end
 
   test do
-    system "#{bin}/spark-shell <<<'sc.parallelize(1 to 1000).count()'"
+    system "echo 'sc.parallelize(1 to 1000).count()' | #{bin}/spark-shell"
```
Comment (dunn) [inline Formula/apache-spark.rb]: You should be able to use our `#pipe_output` method instead; something like:
```ruby
pipe_output("#{bin}/spark-shell", "sc. sc.parallelize(1 to 1000).count()")

Comment (tmozgach) [inline Formula/apache-spark.rb]: I tried different variations, but pipe_output doesn't send command "sc.parallelize(1 to 1000).count()"  inside the prompt of spark-shell. 

When you run spark-shell, the prompt scala> appears and function "sc.parallelize(1 to 1000).count()" should be entered there. 

Any suggestions please?

Comment (dunn) [inline Formula/apache-spark.rb]: Oh, dang.  You might have to use `IO.popen`; [`ircii.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/ircii.rb) is an example.

Comment (tmozgach) [inline Formula/apache-spark.rb]: Sorry, the first proposed option worked. I fixed it! Please merge it!

**Comment by dunn** [comment]
Merged as a17af0056f6d27706bf69503d4d70eb3657a360d, thanks!

**Pull request closed without merging**